server/service/src/sync/translations/name_tag.rs

```diff
@@ -0,0 +1,85 @@
+use repository::{NameTagRow, StorageConnection, SyncBufferRow};
+
+use serde::{Deserialize, Serialize};
+
+use super::{
+    IntegrationRecords, LegacyTableName, PullDeleteRecordTable, PullUpsertRecord, SyncTranslation,
+};
+
+#[allow(non_snake_case)]
+#[derive(Deserialize, Serialize)]
+pub struct LegacyNameTagRow {
+    pub ID: String,
```

Comment:
:nit, This is a little bit opinionated, but I've been suggesting we use serde_rename, and then destructure. The main reason is the reaming quickly shows and in one place, what the translation is ("in one place" argument only really works for remote data, where these is a two way translation)

Reply:
I made a similar comment in another PR, with example of rust analyzer helping, but that was different as renames were already there:
https://github.com/openmsupply/open-msupply/pull/1515#discussion_r1164729509

Reply:
Saw that comment briefly earlier. Will take a closer look definetly keen for consistency :)